speed unit testing.py

```diff
@@ -10,5 +10,6 @@ def driver_speed(speed):
             print ("Licence suspended")
 
 
```

Comment:
here you see the importance of function, if the initial logic was in a function we wouldn't need to duplicate code again here. we would call the function instead